src/se/hms_networks/solutioncenter/logging/Logger.java

```diff
@@ -0,0 +1,271 @@
+package com.hms_networks.americas.sc.logging;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+/**
+ * Logger.java
+ * <p>
+ * Provides an interface for managing log output using log levels.
+ *
+ * @since 1.0
+ * @author HMS Networks, MU Americas Solution Center
+ */
+public class Logger {
+
+  /** Constant for trace log level */
+  public static final int LOG_LEVEL_TRACE = 6;
+
+  /** Constant for debug log level */
+  public static final int LOG_LEVEL_DEBUG = 5;
+
+  /** Constant for info log level */
+  public static final int LOG_LEVEL_INFO = 4;
+
+  /** Constant for warning log level */
+  public static final int LOG_LEVEL_WARN = 3;
+
+  /** Constant for serious log level */
+  public static final int LOG_LEVEL_SER = 2;
+
+  /** Constant for critical log level */
+  public static final int LOG_LEVEL_CRITICAL = 1;

```

Comment:
doesn't this break the convention to go from LOG_LEVEL_SER to LOG_LEVEL_CRITICAL?

Reply:
refactored